### PR TITLE
Pin zeromq to 4.2.*

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/conda-forge.github.io/issues/473

### CODE REVIEW

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -92,6 +92,7 @@
           'tk': 'tk 8.5.*',  # 8.5.18
           'vlfeat': 'vlfeat 0.9.20',  # NA
           'xz': 'xz 5.2.*',  # 5.2.2
+          'zeromq': 'zeromq 4.2.*', # ?
```
Comment (jakirkham) [inline scripts/pin_the_slow_way.py]: @jjhelmus, do you know offhand what version is used in `defaults`?

**Comment by jjhelmus** [comment]
`defaults` is shipping zeromq 4.2.2 and pins pyzmq to `4.2*` except on Windows where the library is bundled.

**Comment by jakirkham** [comment]
Thanks @jjhelmus. Have updated this accordingly.

Also thanks for taking a look @minrk.

**Comment by jakirkham** [comment]
Taking liberties here since it seems we are in agreement.

**Pull request merged**